service/worker/scanner/executions/activities.go

```diff
@@ -361,7 +364,7 @@ func FixShardActivity(
 	for i := heartbeatDetails.LastShardIndexHandled + 1; i < len(params.CorruptedKeysEntries); i++ {
 		currentShardID := params.CorruptedKeysEntries[i].ShardID
 		currentKeys := params.CorruptedKeysEntries[i].CorruptedKeys
-		shardReport, err := fixShard(activityCtx, params, currentShardID, currentKeys, heartbeatDetails)
+		shardReport, err := fixShard(activityCtx, params, currentShardID, currentKeys, heartbeatDetails, params.ScanType)
```

Comment:
looks like params is already passed in

Reply:
good catch! The same thing with the scanShard too. Removed.